pubsub/gossipsub/README.md

```diff
@@ -298,35 +305,29 @@ for each topic in mesh:
      emit PRUNE(topic) control message to peer
 ```
 
-The `fanout` map is maintained by keeping track of last published time
-for each topic:
+The `fanout` map is maintained by keeping track of the length of time since a 
```

Comment:
that's incorrect -- we keep track of the last published time and from that we deduce the length of time since we last published to a topic.